utils/compare_minions.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import salt.config
+import salt.loader
+
+import json
+import json_delta
+
+try:
+    from colorama import Fore, Back, Style, init
+    init()
+except ImportError:  # fallback so that the imported classes always exist
+    class ColorFallback():
+        __getattr__ = lambda self, name: ''
```

Comment:
there is no reason to hack and/or optimize lines of code, so use standard approach instead:
```
def __getattr__(self, name):
    return ''
```